static/configs/service-constants.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Looking for feeback on the format of this file. I have many nested objects inside/maybe that is too awkward to work with?

Reply:
I don't mind the nesting. I'm a bit concerned about exposing the json directly tho. While doing the changes, did TS help you in writing the paths like `Services.ServiceRegistry.Quota.TrialQuotaId`? Onestly this looks like a type to me.